pkg/tools/opa/policies.go

```diff
@@ -77,3 +80,11 @@ func WithTokensExpiredPolicy() *AuthorizationPolicy {
 		checker:      False("tokens_expired"),
 	}
 }
+
+func WithServiceOwnConnectionPolicy() *AuthorizationPolicy {
+	return &AuthorizationPolicy{
+		policySource: tokensServiceConnectionPolicySource,
+		query:        "service_connection",
+		checker:      True("service_connection"),
+	}
+}
```

Comment:
I think we need consider a better name to clarify a fact that the policy should be used only and only with monitors.

Reply:
WithMonitorConnectionServerPolicy